terminus-ui/drawer/src/drawer.component.scss

```diff
@@ -98,6 +99,26 @@ $ts-animation-duration: 400ms;
   &.ts-drawer--end {
     right: 0;
   }
+
```

Comment:
Not related to code in this PR but I must have missed in in the last one.. Background color of the drawer and both headers should be our pure white.

Reply:
This was admittedly a lack of clarity in the issue on my part. Focused on functionality and kind of ignored style 🤦‍♂